### DISCUSSION TITLE
When will the stable version of V3.1.0 be released?

### DISCUSSION THREAD

**Discussion by qianlicao8000** [ColorlibHQ/AdminLTE#3305] [General]
Has a plan?I like the AdminLTE .

**Comment by danny007in** [comment]
Probably this week

**Comment by qianlicao8000** [comment]
Thank you for your offert ,it is a great admin-tamplate.

**Reply by danny007in** [reply]
https://github.com/ColorlibHQ/AdminLTE/graphs/contributors?from=2018-12-06&to=2021-01-13&type=c including them mainly for adminlte v3.x

**Reply by qianlicao8000** [reply]
Thank you guys.

**Comment by qianlicao8000** [comment]
I am waiting.......

**Reply by danny007in** [reply]
little late, because of some bugs.
Now within 3 days u can expect

**Reply by REJack** [reply]
I would say the same, I'm going to release it on sunday if I found no bug or else in the final test ^^.

**Comment by qianlicao8000** [comment]
Thank ou guys,it really a nice project I used.

**Comment by qianlicao8000** [comment]
Oh,no new release means the quality is first....

**Reply by danny007in** [reply]
Exactly,
Because after waiting long time,
If any bug appears,
Then users will get disappointment,
Release is expected in this month,
Or Feb first week.
Thanks 👍🏼

**Reply by qianlicao8000** [reply]
Nice work!

**Comment by qianlicao8000** [comment]
No new information for release?

**Reply by danny007in** [reply]
some issues are there, after that

**Reply by qianlicao8000** [reply]
Can I get a fixed release date?

**Reply by danny007in** [reply]
2 issues must be closed
- https://github.com/ColorlibHQ/AdminLTE/issues/3452
- https://github.com/ColorlibHQ/AdminLTE/issues/3402

**Reply by qianlicao8000** [reply]
How about the bugs fixing?

**Reply by danny007in** [reply]
again one new appered 🤣 
probably for release takes one-two month alteast
